docker-compose.yml

```diff
@@ -52,11 +66,33 @@ services:
       - .:/app:z
     depends_on:
       - redis
-    entrypoint: bash -c 'bundle exec sidekiq'
+    command: bundle exec sidekiq
     environment:
       - MALLOC_ARENA_MAX=2
+      - REDIS_URL=redis://redis:6379
+      - PROMETHEUS_EXPORTER_HOST=prometheus
   redis:
     image: redis:latest
+    ports:
+      - '6379:6379'
+  zookeeper:
+    image: confluentinc/cp-zookeeper
+    environment:
+      - ZOOKEEPER_CLIENT_PORT=32181
+      - ZOOKEEPER_SERVER_ID=1
+  kafka:
```

Comment:
Why not just use the docker-compose kafka and zookeeper from insights-upload? It's kind of out of scope of our project IMO.